pkg/process/tools/dev_setup.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
```

Comment:
`pkg/process/tools` seems like an odd place to put this. Is there anywhere else outside of `pkg` we can move it instead?

Reply:
👍  will also move it under the tools dir